### ISSUE TITLE
validation output during finetuning

### ISSUE THREAD

**Issue by richardzhuang0412** [Lightning-AI/litgpt#1443]
I am doing full finetuning using my custom dataset, every time the validation outputs something like this:
<img width="1170" alt="image" src="https://github.com/Lightning-AI/litgpt/assets/112019253/f79ea760-1dbf-43c9-a38e-bb5843cf1554">
My dataset isn't about movies, is this a bug?

**Comment by rasbt** [comment]
Thanks for reporting. It's not necessarily a bug but more like a hard-coded sanity check. It's to check that the LLM can generate coherent text. We can replace this with a concrete example from the validation set though. I think some other users asked for that a while ago as well.

**Comment by richardzhuang0412** [comment]
Yes please do that it would be really helpful. Or you can add an argument that allow us to select whether we want to see validation output for "recommend a movie" or response to our own dataset. Thanks!
